dts/bindings/base/mutable.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023, Meta
+# SPDX-License-Identifier: Apache-2.0
+
+# Properties for Mutable devices (PM)
+
+properties:
+  mutable:
+    type: boolean
+    description: |
+      True iff the device structure may be mutated.
+
+      Inherit this binding for devices that are runtime-modifiable, in-place.
+      This places the device structure into SRAM rather than Flash.
```

Comment:
this is not hardware description, so at least it should be prefixed with `zephyr,`

Reply:
I don't necessarily see the other draft PR as necessarily blocking this, but I would be happy to minimize whatever might be necessary to fix.

Reply:
Fixed!